### PR TITLE
Add support for rejection emails

### LINKED ISSUE #665: Capability to send rejection email from Vaken
Vaken currently only sends emails by itself when a Hacker Applies, is Accepted, or RSVPs. Add another template for when the Hacker is Rejected.

### PR DESCRIPTION
**Pull request by lucao2018**
Closes #665

### CODE REVIEW

**Comment by gfting** [comment]
The PR looks good, but I would like the following to happen:
- [ ] Do some testing on the `vaken-staging` deployment. I think this would be valuable to make sure it's fine.
- [ ] Blocker: staging is currently broken, I haven't had time to investigate myself.

File: src/server/resolvers/index.ts [hunk 1]
```diff
@@ -318,6 +318,7 @@ export const resolvers: CustomResolvers<Context> = {
 				throw new UserInputError(`user ${id} (${value}) error: ${JSON.stringify(err)}`);
 
 			if (status === ApplicationStatus.Accepted) sendStatusEmail(value, ApplicationStatus.Accepted);
+			if (status === ApplicationStatus.Rejected) sendStatusEmail(value, ApplicationStatus.Rejected);
```
Comment (gfting) [inline src/server/resolvers/index.ts:321]: Even though this should never be a problem, I think that this should be an `else if`

**Comment by BaibhaVatsa** [comment]
iirc @lucao2018 tested this locally and it was working

**Review by leonm1 — APPROVED** [review]

**Comment by leonm1** [comment]
Looks like this is already in `main`, likely through the event branch.

**Pull request closed without merging**